classes/class-list-table.php

```diff
@@ -396,6 +396,12 @@ public function column_default( $item, $column_name ) {
 
 				if ( ! empty( $inserted_columns ) && is_array( $inserted_columns ) ) {
 					foreach ( $inserted_columns as $column_title ) {
+
```

Comment:
@kasparsd or @kidunot89 Correct me if I'm wrong here, but to me this seems like the problem is that the loop needs to exit if there is a value found and also that the default gets set if there isn't a value coming from the filters. 

So the whole code block would change to:
```
if ( ! empty( $inserted_columns ) && is_array( $inserted_columns ) ) {
	foreach ( $inserted_columns as $column_title ) {

		/**
		 * If column title inserted via wp_stream_register_column_defaults ($column_title) exists
		 * among columns registered with get_columns ($column_name) and there is an action associated
		 * with this column, do the action
		 *
		 * Also, note that the action name must include the $column_title registered
		 * with wp_stream_register_column_defaults
		 */
		if ( $column_title === $column_name && has_filter( "wp_stream_insert_column_default_{$column_title}" ) ) {
			/**
			 * Allows for the addition of content under a specified column.
			 *
			 * @param object $record Contents of the row
			 *
			 * @return string
			 */
			$out = apply_filters( "wp_stream_insert_column_default_{$column_title}", $column_name, $record );
			break;
		}
	}
}

// Set default value.
if ( empty( $out ) ) {
	$out = $column_name;
}
```

Reply:
This should work as expected, I will update the PR

One question - If any column value is actually `empty` (for an example case because that column is not relevant for that log type) that gets set to `column_name` everywhere, That is because of the last `// Set default value.` section. Is this an expected behaviour or can that be removed?

Reply:
I’m not sure what the intended default behavior is. Is there a test for this @kidunot89

Reply:
@derekherman @Nikschavan I don't believe so, so I've added some more changes. This should be all ready after somebody else reviews it.